### DISCUSSION TITLE
Javascript if / else if statement

### DISCUSSION THREAD

**Discussion by insomniacj93** [Budibase/budibase#9448] [Help] [answered]
Hello! I'm having trouble with a simple script..

I'm using simple if/else if for translation throughout the apps but i recently run into something:

...
else if ($("Repeater.tb_example.example_row") == null){
	return 'n/a'
}
...

When the row is null, i still don't get the 'n/a', it feels like the script isn't recognising the null value, so i tried with:

if(!$("Repeater.tb_example.example_row")) return 'n/a'

and also

if ($("Repeater.tb_example.example_row") == undefined) {
    return 'n/a';
}

Still same result.. I don't know honestly what i might be missing ^^'

**Comment by melohagan** [comment, marked as answer]
Hey @insomniacj93 

It may be worth adding a Paragraph component underneath your Repeater to see what the value of `$("Repeater.tb_example.example_row")` is.

With that said, I don't think this condition can ever be executed, because if a row didn't exist, then it wouldn't be iterated over.

My guess is that you need to check if `$("Dataprovider.Rows Length")` is empty. Or are you wanting to check if a field in the row is null?

**Reply by insomniacj93** [reply]
I understand what you mean, but the weird part is that this same mechanic (if else statement) I've been applying to the column on the tables and it works fine. If the $("Value") is == null i print out "n/a", same exact script. Only thing i could add as being different besides the fact that I'm printing with a Repeater is that instead of If/else, this time i'm doing if/else if.

**Reply by insomniacj93** [reply]
Got it.. I simply removed the else if (.. == null). Meaning that if none of the previous conditions is used, it will simply print "n/a".

Solution:
```js
...
else if ($("Repeater.tb_example.example_id") == 'something'){
  return 'Something'
}
else {
  return 'n/a'
}
```
